master/internal/kubernetes/pod.go

```diff
@@ -77,6 +79,10 @@ func newPod(
 	}
 	uniqueName := configureUniqueName(msg.Spec)
 
+	// The lifecycle of the containers specified in this map will be monitored.
```

Comment:
where did you put it? I didn't see that anywhere. Have you pushed the change?

Reply:
I mean this comment should be removed and the top-level comment for `pod` should be:
```
// pod manages the lifecycle of a Kubernetes pod that executes a
// Determined task. The lifecycle of the pod is managed based on
// the status of the specified set of containers. A pod will be terminated
// if one of Determined containers exit.
```

Reply:
I saw the comment for `pod` was added in the last time I reviewed but removed now. Was that intentional? 

> I am going to leave the comment here as well, it's important for understanding what this map does.

Shouldn't that comment stay where the field is defined rather than initialized?